store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonSchemaReader.java

```diff
@@ -76,6 +80,17 @@ public static Schema readSchemaInDataFile(String dataFilePath) throws IOExceptio
     return new Schema(columnSchemaList);
   }
 
+  public static String getVersionDetails(String dataFilePath) throws IOException {
```

Comment:
Why add this method in SchemaReader? This information is from Footer, right?

Reply:
yes this info is from footer, in SDK we expose API is schema reader right, i thought may be i can expose one more API there only to get the version details, will this be ok? or need to write this in new class for footer info? please suggest